Isosurfacing_3/examples/Isosurfacing_3/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+# Created by the script cgal_create_cmake_script
+# This is the CMake script for compiling a CGAL application.
+
+cmake_minimum_required(VERSION 3.1...3.14)
+project( Isosurfacing_3_Examples )
+
+find_package(CGAL REQUIRED)
+
+# include for local package
+find_package(Eigen3 3.2.0) #(requires 3.2.0 or greater)
+include(CGAL_Eigen3_support)
+
+create_single_source_cgal_program( "marching_cubes_implicit_sphere.cpp" )
+create_single_source_cgal_program( "marching_cubes_cartesian_grid_sphere.cpp" )
+
+create_single_source_cgal_program( "marching_cubes_mesh_offset.cpp" )
+  target_link_libraries(marching_cubes_mesh_offset PUBLIC CGAL::Eigen3_support)
```

Comment:
all these target link + target creation should be behing `if(TARGET CGAL::Eigen3_support)`

Reply:
Shouldn't `CGAL::Eigen3_support` rather be `REQUIRED`? All examples using Dual Contouring won't compile without Eigen. Or should the Dual Contouring examples just be ignored if Eigen is missing?

Reply:
cartesian gird does not require Eigen for exemple.

Reply:
Even if all required `Eigen`, it would not be `REQUIRED` because you don't want an error when configuring all of CGAL if some third party libs are missing. See https://github.com/CGAL/cgal/blob/6f187f332cbb592c95a250b6a66beadfc5ac8766/Generator/examples/Generator/CMakeLists.txt for an example.

Reply:
Ok, thanks

Reply:
There is also something for OpenMP

Reply:
Yes, that was just to compare the performance of intel tbb with openmp. I will remove that.